TTS/utils/radam.py

```diff
@@ -100,6 +100,9 @@ def step(self, closure=None):
                     if group["weight_decay"] != 0:
                         p_data_fp32.add_(p_data_fp32, alpha=-group["weight_decay"] * group["lr"])
                     p_data_fp32.add_(exp_avg, alpha=-step_size * group["lr"])
+                    torch.save(p_data_fp32, "p_data_fp32.pt")
```

Comment:
Is it necessary to leave this torch.save line to resolve the current issue? I thought this just came from your trial to replicate the issue.

Reply:
adding the torch.save line solves the issue, it makes the code run without any error. It's weird why that works so I opened an issue for it so that you guys can investigate why. for now, I will keep running the code with that patch and get the training results you wanted.

Reply:
i suggest you run the code with and without the torch.save line to confirm yourself. It's incredibly weird :))

Reply:
and beside I also fixed some other error with the training script and added the config file in this PR, the torch.save isnt the only purpose of the PR